airbyte-integrations/connectors/source-paypal-transaction/integration_tests/configured_catalog.json

```diff
@@ -0,0 +1,28 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "transactions",
+        "json_schema": {},
```

Comment:
In one of old source I saw that in configured catalog it is not actually neccessary to specify the whole schema.

Could you please confirm it. Configured catalogs looks more clear with empty json_schema